notifications.json

```diff
@@ -0,0 +1,504 @@
+                    "properties": {
+                        "icon": {
+                            "type": "string"
+                        },
+                        "sound": {
+                            "type": "string"
+                        },
+                        "color": {
+                            "type": "string"
+                        },
+                        "badge": {
+                            "type": "string"
+                        }
+                    }
+                }
+            },
+            "required": [
+                "title"
+            ]
+        },
+        "message-input": {
+            "title": "Message Input",
+            "type": "object",
+            "properties": {
+                "to": {
+                    "description": "One fiscal code (CF).\n",
+                    "type": "string"
+                },
+                "dry_run": {
+                    "type": "boolean",
+                    "description": "This parameter, when set to true, allows developers to test a request without actually sending a message.\n\nThe default value is false."
+                },
+                "time_to_live": {
+                    "type": "integer",
+                    "description": "This parameter specifies how long (in seconds) the message should be kept in storage if the device is offline. The maximum time to live supported is 4 weeks, and the default value is 4 weeks.",
+                    "minimum": 0
+                },
+                "tags": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "content": {
+                    "$ref": "#/definitions/content"
+                }
+            },
+            "required": [
+                "to",
+                "content"
+            ]
+        },
+        "message-status": {
+            "title": "Message Status",
+            "type": "object",
+            "properties": {
+                "status": {
+                    "type": "string",
+                    "enum": [
+                        "PENDING",
+                        "SENDING",
+                        "SENT",
+                        "DELIVERED",
+                        "READ",
+                        "FAILED"
+                    ]
+                },
+                "error_message": {
+                    "type": "string"
+                },
+                "error_code": {
+                    "type": "string",
+                    "enum": [
```

Comment:
I didn't mean the client has to retry. It's an information for the client that there was an error during the delivery but the backend will retry to send the message (causing some delay). Anyway, we can just leave this field open.